cli/src/main/java/com/devonfw/tools/ide/tool/jasypt/Jasypt.java

```diff
@@ -9,19 +9,22 @@
 import com.devonfw.tools.ide.tool.java.Java;
 
 import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Set;
 
 /**
- * {@link ToolCommandlet} for <a href="http://www.jasypt.org/">Jasypt</a>, The java library which allows to add basic
- * encryption capabilities with minimum effort.
+ * {@link ToolCommandlet} for <a href="http://www.jasypt.org/">Jasypt</a>, The java library which allows to add basic encryption capabilities with minimum
+ * effort.
  */
 public class Jasypt extends LocalToolCommandlet {
 
-  public final EnumProperty<JasyptCommand> command;
+  final EnumProperty<JasyptCommand> command;
```

Comment:
We use `public` here since this is the API to the parameters/properties of commendlets.
If one commandlet want to use another commandlet from a different package, you cannot set properties unless they are public (or via `getProperty(String)` method what is not type-safe and would require casting).